src/renderer/store/sharedHelpers.ts

```diff
@@ -15,6 +15,7 @@ export interface ApplicationStore {
   clipboard: Word[];
   // Array of numbers corresponding to indices of words within the transcription
   selection: number[];
+  updateTranscriptionAPIKeyOpened: boolean;
```

Comment:
Not absolutely necessary, but having "update" as the first word could be misread.

Maybe do something like 
`isUpdateTranscriptAPIKeyOpened`